package.json

```diff
@@ -198,6 +204,7 @@
     "lorem-ipsum": "^2.0.3",
     "mini-css-extract-plugin": "^0.9.0",
     "mocha": "^7.1.2",
+    "npm-run-all": "^4.1.5",
```

Comment:
I think my question with regard to this relates to the above mentioned comment :)